data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1479,6 +1479,13 @@ spec:
                   network:
                     description: Network
                     properties:
+                      dnsmasqoptions:
```

Comment:
don't we want to name it ```dnsmasqOptions``` ?

Reply:
yes correct. thanks for that catch. will update